{"nbformat":4,"nbformat_minor":0,"metadata":{"colab":{"name":"walle.ipynb","provenance":[],"collapsed_sections":[]},"kernelspec":{"name":"python3","display_name":"Python 3"},"accelerator":"GPU"},"cells":[{"cell_type":"code","metadata":{"id":"9x6MgHxNPUpw","colab_type":"code","outputId":"b058596c-8815-41e6-afaf-ebd63f96638b","executionInfo":{"status":"ok","timestamp":1592146312105,"user_tz":-120,"elapsed":2976,"user":{"displayName":"ENRIC CAMACHO","photoUrl":"","userId":"08596205074782664455"}},"colab":{"base_uri":"https://localhost:8080/","height":51}},"source":["from google.colab import drive\n","drive.mount('/content/drive')\n","!python3 \"/content/drive/My Drive/DeepLearning_2020/FinalProject/strotss-master/strotss.py\"  \"/content/drive/My Drive/DeepLearning_2020/FinalProject/strotss-master/walle.jpg\" \"/content/drive/My Drive/DeepLearning_2020/FinalProject/strotss-master/style.png\" --weight 1.0 --output walle_face.png --device \"cuda\""],"execution_count":0,"outputs":[{"output_type":"stream","text":["Drive already mounted at /content/drive; to attempt to forcibly remount, call drive.mount(\"/content/drive\", force_remount=True).\n","python3: can't open file '/content/drive/My Drive/DeepLearning_2020/FinalProject/strotss-master/strotss.py': [Errno 2] No such file or directory\n"],"name":"stdout"}]}]}